### PR TITLE
HBASE-24573 do-release should sign the commits and tag it creates

### PR DESCRIPTION
**Pull request by saintstack**

### CODE REVIEW

**Comment by saintstack** [comment]
Testing shows this seems to do the right thing....

```

tag 2.4.1-rm_test2RC8
Tagger: stack <stack@apache.org>
Date:   Tue Dec 22 23:37:31 2020 +0000

via create-release
-----BEGIN PGP SIGNATURE-----

iQIzBAABCgAdFiEEXzLoEtM2TIUEtz4m250xPaeHTykFAl/igzsACgkQ250xPaeH
TynfqA//e9C6a8vZCKWu6MzLDmFuByQpp4rDHC/zogKFx/QT4yHFgdbhGq2bseul
4obQ5by4RWZwtRCQyoLOWzhjnYNM8PAsljzqWIZkrDerpKtta60ADosily4bJWDQ
YaHkbJLrM+tm+RvXfbT24kANu/r9dFztTbA5DOoZ9dHi6YxQYsWP8d+BNPvrNhMC
BFHwgZJAS8guoT0QfI6+zpscfsBbkZxC7D+hDNWqY0rvWvTBgNBDQWlIex3CK4Av
WUct3pWs3qD3igELk9gDYG8dGsdKADLBk+8WiGU7LJtEWk+Ys90gNBjQWEja8DRW
JLQ5YEkWlMdvXoOH4n3KB2c8AYgCWKf53heA/iNIb+rmJBiSQlKtCnD3n3NOeo/m
QGBj8cOXVb4OEIimod+7tvq2nVQQ7fujERqHl/yw5rE7dYlEwIIbEzp2kNXpPaX0
HoE2kllZx8JUoJp67fPtvNsVDydQFx/6cneFXF2ySkhcNlr3tH9cMrlL/otFpUan
fZ+nZTQ0SycLQgtQ8g7Rw5RjNlO+vUBHpzJopse/HemD3ejhr91YGYHqxiXnuvc+
ca2YX0c+T1CdBQZyXi8GCrr8UPevPuceO6GK+p0XdeyOJP9nYSJtuKdlbxwq7Tnj
pPBI+CFZWsMkpFQYvQmAY37waNaSUT3HF4UlJvhY043Tdvr/eeY=
=uQY3
-----END PGP SIGNATURE-----

commit 310e6e12f07e8a25fd74eb4be52b9aeda8287030 (tag: 2.4.1-rm_test2RC8)
Author: stack <stack@apache.org>
Date:   Tue Dec 22 23:37:31 2020 +0000

    Preparing hbase release 2.4.1-rm_test2RC8; tagging and updates to CHANGES.md and RELEASENOTES.md

    Signed-off-by: stack <stack@apache.org>

diff --git a/CHANGES.md b/CHANGES.md
index e7e600ffde..30517642e3 100644
--- a/CHANGES.md
+++ b/CHANGES.md
....
```

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 57s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux 71537019b91d 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 49 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 56s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux 9b3a7b610e3b 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 51 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   2m  7s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux 45b78fd22933 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/1/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 26s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux 05a36fdbe672 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 51 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  4s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 55s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux eee720dad28b 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 55 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 34s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   2m  0s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux adcf53446b85 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 48 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/2/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  9s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   1m 36s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux a268d8cda0b5 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 46 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   1m 59s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux ffe9e04a481a 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 48 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  7s |  Maven dependency ordering for patch  |
||| _ Other Tests _ |
|  |   |   2m  2s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2806 |
| Optional Tests |  |
| uname | Linux ec3dd7c4ea68 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / dcb38f47db |
| Max. process+thread count | 52 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2806/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: dev-support/create-release/release-build.sh [hunk 1]
```diff
@@ -136,19 +136,21 @@ if [[ "$1" == "tag" ]]; then
...
   # Create release version
   maven_set_version "$RELEASE_VERSION"
+  find . -name pom.xml -exec git add {} \;
   git add RELEASENOTES.md CHANGES.md
 
-  git commit -a -m "Preparing ${PROJECT} release $RELEASE_TAG; tagging and updates to CHANGES.md and RELEASENOTES.md"
+  git commit -s -m "Preparing ${PROJECT} release $RELEASE_TAG; tagging and updates to CHANGES.md and RELEASENOTES.md"
   log "Creating tag $RELEASE_TAG at the head of $GIT_BRANCH"
-  git tag "$RELEASE_TAG"
+  git tag -s -m "Via create-release" "$RELEASE_TAG"
```
Comment (ndimiduk) [inline dev-support/create-release/release-build.sh:148]: Maybe better if the message was meaningful to a user looking at the signed tag? Back in 1.1, I used to use "Tag $VERSION $RC", i.e., `git tag -s 1.1.2RC1 -m "Tag 1.1.2RC1"`.

These the only places we commit manually?

Why do we not use the maven release plugin to inc the version number and handle the commits for us? (separate issue)

File: dev-support/create-release/release-build.sh [hunk 2]
```diff
@@ -136,19 +136,21 @@ if [[ "$1" == "tag" ]]; then
 
   git config user.name "$GIT_NAME"
   git config user.email "$GIT_EMAIL"
+  git config user.signingkey "${GPG_KEY}"
 
   # Create release version
   maven_set_version "$RELEASE_VERSION"
+  find . -name pom.xml -exec git add {} \;
```
Comment (ndimiduk) [inline dev-support/create-release/release-build.sh:143]: Explicit add of changed files is A Good Thing, thanks!

**Review by ndimiduk — APPROVED** [review]
A question re: the tag message. Looks good otherwise.

File: dev-support/create-release/release-build.sh (hunk 1 above)
Comment (saintstack) [reply dev-support/create-release/release-build.sh:148]: Your suggestion for the comments (on second thoughts) add no value. We are just adding the tag name as comment; i.e. adding no net info?

Let me combine your suggestion and the original message ; I think it important to note the tag made by the release scripts as opposed to by a human.

Tell me more about mvn release including the version number?

**Comment by saintstack** [comment]
Looks like this went in with

commit 1ca49ea50c8a2b552af9030b4e1bdcd27f618461
Author: Michael Stack <saintstack@users.noreply.github.com>
Date:   Sat Jan 2 11:32:47 2021 -0800

    HBASE-25438 Update create-release mvn in Dockerfile; its 3.6.0; make … (#2807)

... by mistake.

I'll leave it in rather than revert.

Will skip changing the tag message @ndimiduk  unless you think different.

File: dev-support/create-release/release-build.sh (hunk 1 above)
Comment (ndimiduk) [reply dev-support/create-release/release-build.sh:148]: In other projects, I used to use `release:prepare` to apply version change to the poms and create a release tag. A little configuration of the plugin lets you specify these details, it worked pretty well. I was never brave enough to let the plugin push changes though ; i always verified and pushed manually.

Comment (ndimiduk) [reply dev-support/create-release/release-build.sh:148]: https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html

Comment (saintstack) [reply dev-support/create-release/release-build.sh:148]: Oh. That looks useful. HBASE-25466. Thanks.

**Pull request closed without merging**